### PR TITLE
Add Contributor Office Hours

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

File: content/community.html [hunk 1]
```diff
@@ -59,6 +67,10 @@ <h1>Contributing</h1>
       <strong>Development mailing list:</strong>
       <a href="https://groups.google.com/forum/#!forum/prometheus-developers">prometheus-developers</a> (<a href="https://www.mail-archive.com/prometheus-developers@googlegroups.com/">mirror</a>) – for discussions around Prometheus development
     </p>
+    <p>
+      <strong>Office Hours:</strong>
+      <a href="https://docs.google.com/document/d/1bPIsOssTswA244z5SDlws5Qwzc27hQQ2IukNmkIjmPk/edit">Prometheus Contributor Office Hours</a> – public weekly meetings focused on Prometheus development and contributions
```
Comment (bwplotka) [inline content/community.html:72]:
```suggestion
      <a href="https://docs.google.com/document/d/1bPIsOssTswA244z5SDlws5Qwzc27hQQ2IukNmkIjmPk/edit">Prometheus Contributor Office Hours</a> – public weekly meetings focused on Prometheus development and contributions.
```

File: content/community.html [hunk 2]
```diff
@@ -59,6 +67,10 @@ <h1>Contributing</h1>
       <strong>Development mailing list:</strong>
       <a href="https://groups.google.com/forum/#!forum/prometheus-developers">prometheus-developers</a> (<a href="https://www.mail-archive.com/prometheus-developers@googlegroups.com/">mirror</a>) – for discussions around Prometheus development
```
Comment (bwplotka) [inline content/community.html:68]:
```suggestion
      <a href="https://groups.google.com/forum/#!forum/prometheus-developers">prometheus-developers</a> (<a href="https://www.mail-archive.com/prometheus-developers@googlegroups.com/">mirror</a>) – for discussions around Prometheus development.
```

**Review by bwplotka — APPROVED** [review]
Tiny suggestion, otherwise LGTM! 

Thanks 💪🏽

**Comment by roidelapluie** [comment]
mmh those are not sentences

**Comment by bwplotka** [comment]
What they are then? ;p

**Comment by bwplotka** [comment]
Not a biggy, let's merge - fine by me (:

**Comment by roidelapluie** [comment]
And now I see that on the top we used final dots for this. I will make sure to fix this next time I update the page.

**Pull request merged**